content/blog/posts/mps-blog.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Explore Multiplayer Gaming with the O3DE Multiplayer Sample - Now Available for Download!"
```

Comment:
Do we have a Download plan? Do we just want to say "Now Available"?

Reply:
Applied in pending Commit

Reply:
agreed